python/GafferImageUI/CatalogueUI.py

```diff
@@ -51,6 +53,69 @@
 ## then this will probably force us to sort this out. For now, is at least
 ## contained within just this file...
 
+##########################################################################
+# Column Configuration
+#
+# All of the Catalogue's columns are configurable. Columns are identified by
+# a unique key along with a title, value provider and type. Gaffer
```

Comment:
We're still using the `key` terminology here and in a few other places - would be good to get it all lined up.

Reply:
Doh! Sorry, good spot, thanks. Fixed in https://github.com/GafferHQ/gaffer/pull/3676/commits/d1c3a8096f7b00431f50c3b450c61e6a688c615a.